java/org/apache/tomcat/jni/Address.java

```diff
@@ -40,8 +40,9 @@
 
     /**
      * Create apr_sockaddr_t from hostname, address family, and port.
-     * @param hostname The hostname or numeric address string to resolve/parse, or
-     *               NULL to build an address that corresponds to 0.0.0.0 or ::
+     * @param hostname The hostname or numeric address string to resolve/parse, the
```

Comment:
I think the generic name is address which represents `hostname:port`,  `IPv4:port' , `[IPv6]:port`, '/path`. Should we deprecate this one and replace with a new one?

Reply:
This call matches the signature of the APR function, which stayed the same:

http://apr.apache.org/docs/apr/1.6/group__apr__network__io.html#gaa2f399ca2b60b35c0abf7630298c6c9f

My feeling is to keep this aligned, even though "hostname" is technically incorrect.

Reply:
OK, consistency makes sense. I wonder why this has not been addressed in 1.7 or 2.0.